include/cuco/static_map.cuh

```diff
@@ -42,6 +42,23 @@
 
 namespace cuco {
 
+namespace sentinel {
```

Comment:
Probably not the best place to put this. Could we put it in `detail/pair.cuh`?

Reply:
They should be in the public namespace. I'd say putting in its own file `include/cuco/sentinel.hpp`